src/helpers/encodeParams.js

```diff
@@ -4,9 +4,10 @@ const serializeParam = (prefix, param, value) => {
   if (typeof (value) === 'object') {
     return Object.keys(value)
       .map(key => serializeParam(prefixedParam, parseKey(key), value[key]))
+      .filter(serializedParam => !!serializedParam)
       .join('&');
   } else {
-    return [prefixedParam, value].join('=');
+    return value && [prefixedParam, value].join('=');
```

Comment:
Just because I’m seeing a couple of different approaches in the project: What is the preferred way to handle such string concatenations? Using `join`, or using string interpolation like \`${prefixed}=${value}\`? I guess it’s not `+`, right?